### DISCUSSION TITLE
column limit line

### DISCUSSION THREAD

**Discussion by babakabdolahi** [helix-editor/helix#2230] [Q&A] [answered]
Is there a way to specify a column limit line in helix?

**Comment by sudormrfbin** [comment, marked as answer]
There's the `editor.rulers` option implemented in #2060 which draws a vertical line at given column lengths, is that what you mean ? If you meant a column limit to auto hard wrap text as you type then no it doesn't exists yet.

**Comment by babakabdolahi** [comment]
@sudormrfbin yes, that's what I want. Is it included in the stable version? I couldn't find that in the documentation.

**Reply by the-mikedavis** [reply]
https://github.com/helix-editor/helix/pull/2060 was merged after the 22.03 release so it's published to the master docs: https://docs.helix-editor.com/master/configuration.html#editor-section

**Reply by babakabdolahi** [reply]
Thanks
